GoogleForCreators/web-stories-wp
# Support link panel for multi-selection

## Summary

Uses the link value from all selected elements and not just the first.


## Relevant Technical Choices



## To-do




## User-facing changes



## Testing Instructions
I
1. Add 2 elements
2. Add link to one of these
3. Now select both elements
4. Verify that the link's shows now "multiple" in the fields.
5. Change that to some link.
6. Verify that the new value was applied to both elements.

II
1. Add two elements
2. Add link to one of them
3. Now move the element without link to the bottom of the page
4. Add Page Attachment
5. Select both elements and focus on the link input
6. Verify that warning appears saying that link can't be below the dashed line.


---



Fixes #3187
